### DISCUSSION TITLE
can't compile for ESP32S3 using 2.0.10-2.0.14

### DISCUSSION THREAD

**Discussion by hsgentry** [espressif/arduino-esp32#9490] [General]
I have several apps that run on an ESP32S3 and use the LOVYANGFX graphics.  I have always used core 2.0.3, but it is no longer available.  I upgraded to 2.0.14 (I think) and found that it did not work with my app.  Unfortunately I can't get back to 2.0.3.  At this point I am dead in the water.  I will eventually make the newer core work, but I am up against a deadline.  Is there any place that I can get 2.0.3?

**Comment by lyusupov** [comment]
https://github.com/espressif/arduino-esp32/issues/9491#issuecomment-2053660935
